Gemfile

```diff
@@ -47,6 +47,8 @@ group :development do
   gem 'typhoeus'
   gem 'webmock'
 
+  gem 'bigdecimal', '~> 1.3.0'
```

Comment:
fixing for bootstrapping specs but also still works it seems with Ruby 2.x. We cannot use 1.4 of `bigdecimal` due to lack of Ruby 2.0 support.

Reply:
I think it would be safe to drop everything below Ruby 2.4.

- even 2.3 is in EOL for  more than 4 years now
- CI providers stopped supporting everything below 2.4 (Circle CI, Travis CI is about to do that too afaik)

What do you think?

It would make a big difference in upgrading these specs and fixing Ruby 2.7 warnings, especially because Ruby 3.0 will have all warnings and errors.